package org.ezapi.storage.sql;

import java.sql.SQLException;

class Test {

    public static void main(String[] args) throws SQLException {
        /*
        Mysql mysql = new Mysql("127.0.0.1", 3306, "minecraft", "ezapi", "root", "xxxxxxxxxxxx");
        SqlContext sqlContext = new SqlContext();
        sqlContext.setString("test", "1");
        mysql.set("a", sqlContext);
        SqlContext got = mysql.get("a");
        String aaaa = got.getString("test");
        System.out.println(Integer.parseInt(aaaa));
        mysql.close();
        */
        /*
        MongoDB mongoDB = new MongoDB("127.0.0.1", 27017, "root", "xxxxxxxxxxxx", "admin", "test");
        mongoDB.setString("gerry", "key", "I'm value");
        System.out.println(mongoDB.getString("gerry", "key"));
        */
    }

}
